pywhy_graphs/algorithms/tests/test_cpdag.py

```diff
@@ -0,0 +1,243 @@
+import networkx as nx
+import numpy as np
+import pytest
+
+import pywhy_graphs.networkx as pywhy_nx
+from pywhy_graphs.algorithms import all_vstructures
+from pywhy_graphs.algorithms.cpdag import (
+    EDGELABELS,
+    dag_to_cpdag,
+    label_edges,
+    order_edges,
+    pdag_to_cpdag,
+    pdag_to_dag,
+)
+from pywhy_graphs.testing import assert_mixed_edge_graphs_isomorphic
+
+seed = 12345
+rng = np.random.default_rng(seed)
+
+
+class TestOrderEdges:
+    def test_order_edges_errors(self):
+        G = nx.DiGraph()
+
+        # 1 -> 2 -> 4 -> 5
+        # 1 -> 3 -> 4
+        # so topological sort is: (1, 2, 3, 4, 5)
+        G.add_edges_from([(1, 2), (1, 3), (2, 4), (3, 4), (4, 5)])
+        # now test when there is a cycle
+        G.add_edge(5, 1)
+        with pytest.raises(ValueError, match="G must be a directed acyclic graph"):
+            order_edges(G)
+
+    def test_order_edges(self):
+        # Example usage:
+        G = nx.DiGraph()
+
+        # 1 -> 2 -> 4 -> 5
+        # 1 -> 3 -> 4
+        G.add_edges_from([(1, 2), (1, 3), (2, 4), (3, 4), (4, 5)])
+        G = order_edges(G)
+
+        print("Edge labels:", G.edges.data())
+        expected_order = [
+            (1, 2, {"order": 4}),
+            (1, 3, {"order": 3}),
+            (2, 4, {"order": 1}),
+            (3, 4, {"order": 2}),
+            (4, 5, {"order": 0}),
+        ]
+        assert set(G.edges.data(data="order")) == set(
+            [(src, target, order["order"]) for src, target, order in expected_order]
+        )
+
+        # Add a string as a node
+        # 5 -> 3 -> 1 -> 2 -> 'a'; 1 -> 'b'
+        G = nx.DiGraph()
+        G.add_edges_from([(5, 3), (3, 1), (1, 2), (2, "a"), (1, "b")])
+        G = order_edges(G)
+        print("Edge labels:", G.edges.data())
```

Comment:
I forget - do we have a convention of stripping prints from test files in this project?

Reply:
No, but I will remove these print statements for the sake of cleanness.